python/craftassist/shapes.py

```diff
@@ -26,7 +26,7 @@ def rectangle(
     bid: IDM = DEFAULT_IDM,
     orient="xy",
     labelme=False,
-    **kwargs,
+    **kwargs
```

Comment:
This is probably a formatting thing from your editor and isn't a functional change.